### ISSUE TITLE
Wording "Appointments" -> "Appointment booking"

### ISSUE THREAD

**Issue by bentuna** [nextcloud/calendar#6552]
### Is your feature request related to a problem? Please describe.

In English as well as in German people are confused by the name of the Appointments functionality. When they first see it, they don't know what it does.

### Describe the solution you'd like

I would suggest

| Use Case | Now | Suggestion EN | Suggestion DE
|--------|--------|--------|--------|
| name of the functionality, Title in the calendar sidebar | Appointments | Appointment booking | Terminbuchung |
| title of the modal "Create/Edit ..." | Create/Edit appointment | Create/Edit appointment schedule | Buchungsplan erstellen/bearbeiten |
| label of link to booking page | Preview | booking page | Buchungsseite |

**Comment by SebastianKrupinski** [comment]
@nimishavijay @marcoambrosini what do you think?

**Comment by nimishavijay** [comment]
Good point! IIRC this isn't the first issue regarding the wording, so it can definitely be improved :)

Mein Deutsch ist nicht so gut so I'm gonna stick with English ;) 

 > Create/Edit appointment schedule 

This is perfect, we had this discussed in a design review call as well

> Appointment booking

I would just go with "Appointment schedule" to keep things consistent

> Booking page

This also sounds good, I would suggest that it be "View booking page" as it is an action that opens the page in a new tab.

**Comment by bentuna** [comment]
> Good point! IIRC this isn't the first issue regarding the wording, so it can definitely be improved :)
> 
> Mein Deutsch ist nicht so gut so I'm gonna stick with English ;)
> 
> > Create/Edit appointment schedule
> 
> This is perfect, we had this discussed in a design review call as well
> 
> > Appointment booking
> 
> I would just go with "Appointment schedule" to keep things consistent
> 
> > Booking page
> 
> This also sounds good, I would suggest that it be "View booking page" as it is an action that opens the page in a new tab.

Sounds all good to me! 👍
